code/game/objects/items/pillow.dm

```diff
@@ -7,6 +7,7 @@
 	inhand_icon_state = "pillow_t"
 	lefthand_file = 'icons/mob/inhands/items/pillow_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/items/pillow_righthand.dmi'
+	force = 5
 	w_class = WEIGHT_CLASS_NORMAL
 	damtype = STAMINA
```

Comment:
Wouldn't this need to change damtype when getting bricked?

Reply:
I intentionally left the damtype as stamina when bricked because the pillow fluff doesn't get removed, you're just putting a brick in the middle.
Also I just personally would prefer bricked pillows to not be a lame brute damage weapon (the game is full of those!) and instead just be a lame stamina damage weapon.